### PR TITLE
Automl beta dataset

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

**Comment by nnegrey** [comment]
Some beta samples haven't been merged to master and are on branches. 
This PR takes the published V1 samples and copies them over to the beta folder for the products still in GA.

**Comment by nnegrey** [comment]
Partial fix for : https://github.com/GoogleCloudPlatform/java-docs-samples/issues/2011

File: automl/beta/src/main/java/com/example/automl/ExportDataset.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import com.google.cloud.automl.v1beta1.AutoMlClient;
+import com.google.cloud.automl.v1beta1.DatasetName;
+import com.google.cloud.automl.v1beta1.GcsDestination;
+import com.google.cloud.automl.v1beta1.OutputConfig;
+import com.google.protobuf.Empty;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+class ExportDataset {
```
Comment (nnegrey) [inline automl/beta/src/main/java/com/example/automl/ExportDataset.java]: Delete export, not used in docs

File: automl/beta/src/main/java/com/example/automl/GetDataset.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+// [START automl_video_classification_get_dataset_beta]
+import com.google.cloud.automl.v1beta1.AutoMlClient;
+import com.google.cloud.automl.v1beta1.Dataset;
+import com.google.cloud.automl.v1beta1.DatasetName;
+
+import java.io.IOException;
+
+class GetDataset {
+
+  static void getDataset() throws IOException {
```
Comment (nnegrey) [inline automl/beta/src/main/java/com/example/automl/GetDataset.java]: Delete, not in docs

**Comment by lesv** [comment]
admin merge - all green

**Pull request merged**